card/db/schema.rb

```diff
@@ -66,20 +66,20 @@
   add_index "card_revisions", ["creator_id"], name: "revisions_created_by_index", using: :btree
 
   create_table "cards", force: :cascade do |t|
-    t.string   "name",                limit: 255,   null: false
-    t.string   "key",                 limit: 255,   null: false
-    t.string   "codename",            limit: 255
     t.integer  "left_id",             limit: 4
-    t.integer  "right_id",            limit: 4
-    t.integer  "current_revision_id", limit: 4
     t.datetime "created_at",                        null: false
     t.datetime "updated_at",                        null: false
+    t.integer  "current_revision_id", limit: 4
+    t.string   "name",                limit: 255,   null: false
```

Comment:
I get that, but it's still curious to me, because it's based on the column order in the db, which is, in turn, based on a previous copy of this file.  So I wonder how your db got the weird order?